lambda/src/main/resources/db/migration/V73__rename-closed_for-to-closed-and-open_on_transfer-to-open.sql

```diff
@@ -0,0 +1,13 @@
+-- Change FilePropertyValues.ClosureType values
```

Comment:
The migration version number in the file name will need to be updated. The latest number is `V75`. So this will need to be `V76`

Reply:
I'm seeing V71, V72 and V73 as available in the main branch. I'm happy to go to V76 if that's the best thing to do. Could you confirm?

Reply:
Oh, yes, quite correct, I just saw "75" and assumed lower version were used.